src/math/Kconfig

```diff
@@ -22,15 +22,35 @@ config POWER_FIXED
 
 
 config BINARY_LOGARITHM_FIXED
-       bool "Binary Logarithm function"
-       default n
-       help
```

Comment:
this should be a different commit

Reply:
or even better: not in this Pull Request. Or maybe even better: not in any Pull Request. It looks like pure git conflict material.

Reply:
It does look like a valid change though as it is a conversion to tabs

Reply:
Valid yes but useful?

Reply:
Reverted to original state. No code